sources/osg/KdTree.js

```diff
@@ -121,7 +121,7 @@ BuildKdTree.prototype = {
             if ( mode === PrimitiveSet.TRIANGLES )
                 totalLenArray += prim.getCount();
             else if ( mode === PrimitiveSet.TRIANGLE_STRIP || mode === PrimitiveSet.TRIANGLE_FAN )
-                totalLenArray += ( prim.getCount() - 2 ) * 3;
+                totalLenArray += Math.abs( ( prim.getCount() - 2 ) ) * 3;
```

Comment:
Would have be good to have this in a separate commit for the history because it's clearly not related to webgl2